torch/quantization/__init__.py

```diff
@@ -1,15 +1,147 @@
-from .quantize import *  # noqa: F403
```

Comment:
I'm not sure such changes are desired. For me such usages of `import *` is fine.

Otherwise one needs to change the code in two places every time and hundreds of lines of code needlessly get added to the codebase.

Reply:
good point.

but what I worry about is that the namespace gets polluted with random names. Some of the names mentioned in quantize.py are pretty short and generic. If you feel like it'll be ok I can turn it back.

Reply:
:+1: